src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -601,6 +602,20 @@ public TypeSymbolWithAnnotations SetUnknownNullabilityForReferenceTypesIfNecessa
                 this.SetUnknownNullabilityForReferenceTypes();
         }
 
+        // PROTOTYPE(NullableReferenceTypes): external annotations should be removed or fully designed/productized
+        public TypeSymbolWithAnnotations SetUnknownNullabilityForReferenceTypesIfNecessary(ModuleSymbol module, ImmutableArray<bool> extraAnnotations)
```

Comment:
>extraAnnotations [](start = 133, length = 16)

The name of this parameter feels confusing. It looks like these are not extra annotations, but the nullable annotations to use.